sklearn/model_selection/_validation.py

```diff
@@ -231,6 +287,10 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     X_train, y_train = _safe_split(estimator, X, y, train)
     X_test, y_test = _safe_split(estimator, X, y, test, train)
 
+    n_scorers = len(scorers.keys())
+    test_scores = dict(zip(scorers.keys(), [error_score, ] * n_scorers))
```

Comment:
Sorry could you clarify a bit? Do you mean to say, why not return a list and let the calling function make it into a dict?